common/Services/PackageDeploymentService.cs

```diff
@@ -31,4 +34,49 @@ public async Task RegisterPackageForCurrentUserAsync(string packageFamilyName, R
             throw new RegisterPackageException(result.ErrorText, result.ExtendedErrorCode);
         }
     }
+
+    /// <inheritdoc />
+    public IEnumerable<Package> FindPackagesForCurrentUser(string packageFamilyName, Version minVersion1, Version version1Ceiling, Version minVersion2)

```

Comment:
IIRC `Package` is a projected class type and I am not sure how to mock it for testing purposes. So that may be an issue in the future if a unit test was attempting to return dummy package results for a unit test. No action for this PR. Resolving.